src/Resources/contao/library/Contao/StringUtil.php

```diff
@@ -511,31 +511,64 @@ public static function toHtml5($strString)
 	 * @return string The converted string
 	 *
 	 * @throws \Exception If $strString cannot be parsed
+	 * @throws \InvalidArgumentException If $strString or any token in $arrData contains php code
 	 */
 	public static function parseSimpleTokens($strString, $arrData)
 	{
 		$strReturn = '';
 
-		// Remove any unwanted tags (especially PHP tags)
-		$strString = strip_tags($strString, \Config::get('allowedTags'));
+		// Do not allow php code for security reasons.
+		$containsPhp = function($value) {
+			return strpos($value, '<?php') !== false
+				|| strpos($value, '<?') !== false
+				|| strpos($value, '<%') !== false
+				|| strpos($value, '<%=') !== false
```

Comment:
Looks like `<script foo="bar" language="php">` doesn’t get executed. But `<script  language  =  PHP  >` does for example (in PHP 5)